Tests/MongoSwiftTests/ReadWriteConcernTests.swift

```diff
@@ -30,19 +30,19 @@ final class ReadWriteConcernTests: MongoSwiftTestCase {
     func testReadConcernType() throws {
         // check level var works as expected
         let rc = ReadConcern(.majority)
-        expect(rc.level).to(equal("majority"))
+        expect(rc.level).to(equal(ReadConcern.Level.majority))
 
         // test copy init
         let rc2 = ReadConcern(from: rc)
-        expect(rc2.level).to(equal("majority"))
+        expect(rc2.level).to(equal(ReadConcern.Level.majority))
 
         // test empty init
         let rc3 = ReadConcern()
         expect(rc3.level).to(beNil())
```

Comment:
I know this is existing code, but I think it makes sense to add a test for `isDefault` here.